modules/serverless-cluster-sizing-req.adoc

```diff
@@ -9,3 +14,5 @@ The following requirements relate only to the pool of worker machines of the {pr
 ====
 
 The total size requirements to run {ServerlessProductName} are dependent on the applications deployed. By default, each pod requests approximately 400m of CPU, so the minimum requirements are based on this value. Lowering the actual CPU request of applications can increase the number of possible replicas.
+
+If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200MB - 2GB of memory for each replica of the Knative Serving control plane.
```

Comment:
They were provided a long time ago when we first added this section I think - it's showing as green here because I moved it from the assembly to this module, but I didn't change the content 🙂 we have a Jira open to revise the requirements for Serverless in general I think, but are waiting for testing from the performance team afaik.